src/mono/wasm/Wasm.Build.Tests/Blazor/BuildPublishTests.cs

```diff
@@ -276,4 +276,39 @@ private void BlazorAddRazorButton(string buttonText, string customCode, string m
         string oldContent = File.ReadAllText(counterRazorPath);
         File.WriteAllText(counterRazorPath, oldContent + additionalCode);
     }
+
+    [Theory, TestCategory("no-workload")]
```

Comment:
Why only `no-workload`?

Reply:
Copy paste thing. I'll remove it

Reply:
@radical Any idea why it fails now https://[helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-89600-merge-515cee0e0ad8479bb3/Workloads-NoWebcil-Wasm.Build.Tests.Blazor.BuildPublishTests/1/console.6de06935.log?helixlogtype=result](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-89600-merge-515cee0e0ad8479bb3/Workloads-NoWebcil-Wasm.Build.Tests.Blazor.BuildPublishTests/1/console.6de06935.log?helixlogtype=result) ?

Reply:
It complains about unexpected folder `obj/Debug/net8.0/wasm/for-build`

Reply:
Is that because of the build and than publish?